tensorflow/contrib/losses/python/losses/loss_ops.py

```diff
@@ -104,7 +78,9 @@ def _safe_mean(losses, num_present):
       then zero is returned.
   """
   total_loss = math_ops.reduce_sum(losses)
-  return _safe_div(total_loss, num_present)
+  return math_ops.div_no_nan(total_loss,
+                             math_ops.maximum(num_present, 0),
```

Comment:
I think `maximum` is unnecessary op here, because num_present should be non-negative in practice.

Reply:
Does something break when you remove the maximum? This could be another case of us needing the double-tap-where trick to avoid NaNs in the gradient.

Reply:
Hi, I don't remove the maximum. Test failed after I resolved the conflict with the lastest master. I'll tell you when I fix it later. :-)